### PR TITLE
add exportRender docs

### PR DESCRIPTION
**Pull request by mrezagolestan**
this commit add document for exportRender method which is presented for render column for print or export as we expected

this method added via two commits:
d48dc1bbe8bc860070240d6f1444e7a861c53789 commit in laravel-datatables-html via pull request #195
56b94a4482f4273ce57b08b5cd05dbc695dfce22 commit in laravel-datatables-buttons via pull request #177

### CODE REVIEW

**Comment by mrezagolestan** [comment]
please release this too ( mentioned #195 & #177 released) for doc

File: html-builder-column-builder.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ $column = Column::make('id')
         ->searchable(true)
         ->orderable(true)
         ->render('\'<div class="editor-active" >\' + (full[\'deleted_at\'] == null ? \'<i class="fas fa-check-circle client-is-active"></i>Active\' : \'<i class="fas fa-times-circle"></i>Inactive\') + \'</div>\';\'\'' )
+        ->render(function($row,$data){return $data == 1 ? 'Active' : 'Inactive'})
```
Comment (yajra) [inline html-builder-column-builder.md:34]: I think this is not needed.

**Review by yajra — CHANGES_REQUESTED** [review]

**Comment by mrezagolestan** [comment]
yep. it is exportRender, i changed it

**Comment by yajra** [comment]
Thanks for the contributions, appreciate it! 🍻

**Pull request merged**